Закончил файл bounty.config.json

vlad8599 opened the pull request:
Описание говорит само за себя, проверьте, пожалуйста.

vlad8599 committed:
Merge 17ff8f11ec9999d0db9eef94ed17fdd9fc08f5ae into web-interface

vlad8599 committed:
Merge a45e08e2650e47239b3212ce44f588fee04e103d into web-interface

vlad8599 committed:
Merge 91b0bdabd270a218b6548d69ef1b8e29f6d9c8fa into web-interface

vlad8599 committed:
Обновление translations/texts/dialog/bounty.config.json

vlad8599 committed:
Обновление translations/texts/dialog/bounty.config.json

vlad8599 committed:
Обновление translations/texts/dialog/bounty.config.json

vlad8599 committed:
Merge 4612c7374eac52c22667aca4e57eb8eccc1816a9 into web-interface

vlad8599 committed:
Обновление translations/texts/dialog/bounty.config.json

vlad8599 committed:
Обновление translations/texts/dialog/bounty.config.json

vlad8599 committed:
Обновление translations/texts/dialog/bounty.config.json

vlad8599 committed:
Закончил файл bounty.config.json

xomachine commented on translations/texts/dialog/bounty.config.json:
```diff
@@ -649,7 +677,8 @@
     "Texts": {
-      "Eng": "I am merely a wandering bard. I am not worthy of any suspicion! Verily!"
+      "Eng": "I am merely a wandering bard. I am not worthy of any suspicion! Verily!",
+      "Rus": "Я просто странствующий бард. Я не достоин каких-либо подозрений! Воистину!"
```
Тут, похоже речь идёт не столько о том, что он не достоин, а о том что не стоит тратить на него время, тк в нём нет ничего подозрительного.
Не достоин звучит как будто он слишком плох чтобы его подозревать

xomachine commented on translations/texts/dialog/bounty.config.json:
```diff
@@ -660,7 +689,8 @@
     "Texts": {
-      "Eng": "I am merely a wandering nomad. I am not worthy of any suspicion!"
+      "Eng": "I am merely a wandering nomad. I am not worthy of any suspicion!",
+      "Rus": "Я просто странствующий кочевник. Я не достоин каких-либо подозрений! Воистину!"
```
то же, что и выше

xomachine commented on translations/texts/dialog/bounty.config.json:
```diff
@@ -695,7 +727,8 @@
     "Texts": {
-      "Eng": "I believe I'm in quite a predicament, yesiree!"
+      "Eng": "I believe I'm in quite a predicament, yesiree!",
+      "Rus": "Я уверен, что я в затруднительном положении, да!"
```
I believe тут больше похоже на "я думаю", чем на "я уверен"

xomachine commented on translations/texts/dialog/bounty.config.json:
```diff
@@ -765,7 +803,8 @@
     "Texts": {
-      "Eng": "I guess I'm not fast enough - I thought these feathers would make me more aerodynamic!"
+      "Eng": "I guess I'm not fast enough - I thought these feathers would make me more aerodynamic!",
+      "Rus": "Думаю, я недостаточно быстр - я думал, что эти перья сделают меня более аэродинамичным!"
```
Думаю наверное лучше заменить на "полагаю", чтобы не было повтора

xomachine commented on translations/texts/dialog/bounty.config.json:
```diff
@@ -816,7 +858,8 @@
     "Texts": {
-      "Eng": "I have utilised fake handcuffs to trick you! I am still free!"
+      "Eng": "I have utilised fake handcuffs to trick you! I am still free!",
+      "Rus": "Я выкинул фальшивые наручники, чтобы обмануть тебя! Я все еще свободен!"
```
скорее использовал<self.pronoun.verbEnding>

xomachine commented on translations/texts/dialog/bounty.config.json:
```diff
@@ -838,7 +882,8 @@
     "Texts": {
-      "Eng": "I knew I could buy you out! Thanks for helping me get away!"
+      "Eng": "I knew I could buy you out! Thanks for helping me get away!",
+      "Rus": "Я знал, что могу выкупить твою дою! Спасибо, что помог мне выбраться!"
```
подкупить тебя + окончания не хватает у "знал" и у "помог"

xomachine commented on translations/texts/dialog/bounty.config.json:
```diff
@@ -853,7 +898,8 @@
     "Texts": {
-      "Eng": "I knew I could buy you out! Thanks for helping me get away, pal!"
+      "Eng": "I knew I could buy you out! Thanks for helping me get away, pal!",
+      "Rus": "Я знал, что могу выкупить твою дою! Спасибо, что помог мне выбраться, приятель!"
```
см выше

xomachine commented on translations/texts/dialog/bounty.config.json:
```diff
@@ -864,7 +910,8 @@
     "Texts": {
-      "Eng": "I knew you looked like the type who would accept a bribe!"
+      "Eng": "I knew you looked like the type who would accept a bribe!",
+      "Rus": "Я знал, что ты выглядишь как тип, который согласится на взятку!"
```
не хватает окончания, тк мы не знаем пол говорящего

знал<self.pronoun.verbEnding>

кроме того, не очень хорошо тут подходит слово тип. неплохо бы перефразировать

xomachine commented on translations/texts/dialog/bounty.config.json:
```diff
@@ -919,7 +970,8 @@
     "Texts": {
-      "Eng": "I remember now what we were made for! THEY BUILT US TO MEET ^orange;IT^reset;! DON'T YOU SEE?!"
+      "Eng": "I remember now what we were made for! THEY BUILT US TO MEET ^orange;IT^reset;! DON'T YOU SEE?!",
+      "Rus": "Теперь я помню, для чего мы были созданы! ОНИ СОЗДАЛИ, ЧТОБЫ ВСТРЕТИТЬ ^orange;ЭТО^reset;! НЕ ВИДИШЬ ?!"
```
создали кого?

xomachine commented on translations/texts/dialog/bounty.config.json:
```diff
@@ -941,7 +994,8 @@
     "Texts": {
-      "Eng": "I see no criminals around here, Peacekeeper. Not a single bounty in sight."
+      "Eng": "I see no criminals around here, Peacekeeper. Not a single bounty in sight.",
+      "Rus": "Здесь я не вижу преступников, Миротворец. Ни одной награды вокруг."
```
награда тут не слишком подходит, надо перефразировать

xomachine commented on translations/texts/dialog/bounty.config.json:
```diff
@@ -955,7 +1009,8 @@
     "Texts": {
-      "Eng": "I see no criminals around here. No siree, not a bounty in sight."
+      "Eng": "I see no criminals around here. No siree, not a bounty in sight.",
+      "Rus": "Я не вижу ни одного преступника вокруг. Нет, сэр, ни одной награды в поле зрения."
```
см выше

xomachine commented on translations/texts/dialog/bounty.config.json:
```diff
@@ -966,7 +1021,8 @@
     "Texts": {
-      "Eng": "I shouldn't have volunteered for all those Miniknog test programs... I've never been the same since!"
+      "Eng": "I shouldn't have volunteered for all those Miniknog test programs... I've never been the same since!",
+      "Rus": "Я не должен был добровольно участвовать во всех этих тестах Миникнога... Я никогда не буду прежним!"
```
Ой-ой. Согласно глоссарию, никакого Миникнога у нас нет, есть только Министерство

кроме того, речь идёт о прошедшем времени. предлагаю использовать фразу
"меня будто подменили с тех пор" чтобы согласовать этот перевод с другими

ещё не хватает окончания, тк нам неизвестен пол говорящего

xomachine commented on translations/texts/dialog/bounty.config.json:
```diff
@@ -988,7 +1045,8 @@
     "Texts": {
-      "Eng": "I sure ain't up to any wrong-doin' today! I'm jus' here to... Paint my wagon!"
+      "Eng": "I sure ain't up to any wrong-doin' today! I'm jus' here to... Paint my wagon!",
+      "Rus": "Я уверен, что не сделал ничего плохого сегодня! Я здесь просто, чтобы ... покрасить свой фургон!"
```
не хватает окончания
советую представлять что каждую фразу говорит как женский персонаж, так и мужской, чтобы не допускать подобных ошибок

xomachine commented on translations/texts/dialog/bounty.config.json:
```diff
@@ -827,7 +870,8 @@
     "Texts": {
-      "Eng": "I installed some new parts in my brain and now I can see the secret colour!"
+      "Eng": "I installed some new parts in my brain and now I can see the secret colour!",
+      "Rus": "Я установил некоторые новые части в мой мозг и теперь я могу видеть тайные цвета!"
```
нет окончания

xomachine commented on translations/texts/dialog/bounty.config.json:
```diff
@@ -805,7 +846,8 @@
     "Texts": {
-      "Eng": "I have realised that clothes only bind us - but I am a slave no more!"
+      "Eng": "I have realised that clothes only bind us - but I am a slave no more!",
+      "Rus": "Я понял, что одежда нас только сковывает - но я больше не раб!"
```
окончание

xomachine commented on translations/texts/dialog/bounty.config.json:
```diff
@@ -779,7 +818,8 @@
     "Texts": {
-      "Eng": "I guess I'm not fast enough. Does that mean we have to fight, instead?"
+      "Eng": "I guess I'm not fast enough. Does that mean we have to fight, instead?",
+      "Rus": "Похоже, что я слишком медленный. Значит ли это, что мы должны драться?"
```
окончание

xomachine commented on translations/texts/dialog/bounty.config.json:
```diff
@@ -1078,7 +1143,8 @@
     "Texts": {
-      "Eng": "I'm not a criminal - I'm just... Another travelling Avian, spreading the word of Kluex! Praise be!"
+      "Eng": "I'm not a criminal - I'm just... Another travelling Avian, spreading the word of Kluex! Praise be!",
+      "Rus": "Я не преступник - я просто ... еще один путешествующий Авиан, распространяющий слово Клюэкса! Хвала!"
```
несущий будет лучше

Ну и слово Хвала как-то в воздухе подвешено получилось. лучше пусть будет Хвала ему!

xomachine commented on translations/texts/dialog/bounty.config.json:
```diff
@@ -1100,7 +1167,8 @@
     "Texts": {
-      "Eng": "I've been rumbled, and by a Floran no less!"
+      "Eng": "I've been rumbled, and by a Floran no less!",
+      "Rus": "Я был ошарашен, и флоран не меньше!"
```
Честно говоря, я бы не стал это переводить так.
Наверно тут говорится "и Флораном не меньше", но без контекста я бы вообще пока не стал это переводить

xomachine commented on translations/texts/dialog/bounty.config.json:
```diff
@@ -1377,7 +1468,8 @@
     "Texts": {
-      "Eng": "Like a leaf in a river, I flee!"
+      "Eng": "Like a leaf in a river, I flee!",
+      "Rus": "Как лист в реке, я ускользаю!"
```
по реке будет лучше звучать

xomachine commented on translations/texts/dialog/bounty.config.json:
```diff
@@ -1577,7 +1685,8 @@
     "Texts": {
-      "Eng": "The secret truth is that everything is pixelly and flat! It can't just be me who sees it, can it?!"
+      "Eng": "The secret truth is that everything is pixelly and flat! It can't just be me who sees it, can it?!",
+      "Rus": "Секрет в том, что все пиксельно и плоско! Я не могу быть всего лишь тем, кто это видит, не так ли ?!"
```
Не может быть, чтобы лишь только я это видел<self.pronoun.verbEnding>

xomachine commented on translations/texts/dialog/bounty.config.json:
```diff
@@ -1593,7 +1702,8 @@
     "Texts": {
-      "Eng": "There you go! I knew I could bribe my way out of this situation. See you around!"
+      "Eng": "There you go! I knew I could bribe my way out of this situation. See you around!",
+      "Rus": "Вот и ты! Я знал, что смогу подкупить свой выход из этой ситуации. Увидимся!"
```
Подкупить можно человека, но не выход. Лучше просто купить

xomachine commented on translations/texts/dialog/bounty.config.json:
```diff
@@ -1651,7 +1765,8 @@
     "Texts": {
-      "Eng": "Time for Floran to run!"
+      "Eng": "Time for Floran to run!",
+      "Rus": "Время флорану бежать!"
```
Флорану пора бежать!

xomachine commented on translations/texts/dialog/bounty.config.json:
```diff
@@ -1785,7 +1910,8 @@
     "Texts": {
-      "Eng": "You Floran are meant to be dumb enough to fall for that!"
+      "Eng": "You Floran are meant to be dumb enough to fall for that!",
+      "Rus": "Флоран, ты был настолько туп, чтобы вляпаться в это!"
```
fall for something - купиться на что-то
ну и "туп" лучше заменить на "глуп"

xomachine reviewed (commented):
Чего тут не хватает, так это окончаний. Кое-где я отметил, но таких мест слишком много.
Рекомендую сделать вычитку всего этого представляя говорящего и игрока как в женском роде, так и в мужском

vlad8599 commented:
Хорошо, постараюсь исправить с родами.

vlad8599 committed:
Merge 0af9f95f00a794aff603db5ce5115e24d6289d31 into web-interface

vlad8599 committed:
Обновление translations/texts/dialog/bounty.config.json

vlad8599 committed:
Обновление translations/texts/dialog/bounty.config.json

vlad8599 commented:
Исправил все, перевод фразы про "Я был ошарашен" временно удалил.

xomachine commented:
> Исправил все, перевод фразы про "Я был ошарашен" временно удалил.

Почему-то я вижу лишь пару исправлений

vlad8599 commented:
Мда, очешуеть. Гитхаб нормально не сохранил исправления почему-то. Блин, все по новой...

xomachine commented:
Такое бывает, если в интерфейсе нажимать несколько раз подряд кнопку сохранить. Баг плавающий и не всегда воспроизводится, потому отловить его мне не удалось

vlad8599 committed:
Повторные исправления(предыдущие не сохранились)

vlad8599 commented:
Так щас вроде стало лучше. Но черт возьми, здесь обновляется, а у меня в личной копии изменения не сразу отображаются! Какого черта?!

xomachine commented on translations/texts/dialog/bounty.config.json:
```diff
@@ -502,7 +502,7 @@
     "Texts": {
       "Eng": "Good choice. I knew you looked like the type who would accept a bribe!",
-      "Rus": "Хороший выбор. Я знал, что ты выглядишь как тип, который согласится на взятку!"
+      "Rus": "Хороший выбор. Я знал, что ты смахиваешь на кого-то, кто согласится на взятку!"
```
знал<>

xomachine commented on translations/texts/dialog/bounty.config.json:
```diff
@@ -569,7 +569,7 @@
     "Texts": {
       "Eng": "Haha, you haven't actually caught me. Those were fake handcuffs I put on!",
-      "Rus": "Хаха, ты еще не поймал меня. Это были фальшивые наручники, которые я одел!"
+      "Rus": "Хаха, ты еще не поймал<player.pronoun.verbEnding>  меня. Это были фальшивые наручники, которые я одел<self.pronoun.verbEnding> !"
```
надел<>

xomachine commented on translations/texts/dialog/bounty.config.json:
```diff
@@ -557,7 +557,7 @@
     "Texts": {
       "Eng": "Haha, you haven't actually caught me. Those were fake handcuffs I put on myself!",
-      "Rus": "Хаха, ты еще не поймал меня. Это были фальшивые наручники, которые я на себя одел!"
+      "Rus": "Хаха, ты еще не поймал<player.pronoun.verbEnding>  меня. Это были фальшивые наручники, которые я на себя одел<self.pronoun.verbEnding> !"
```
надел<>

xomachine commented on translations/texts/dialog/bounty.config.json:
```diff
@@ -678,7 +678,7 @@
     "Texts": {
       "Eng": "I am merely a wandering bard. I am not worthy of any suspicion! Verily!",
-      "Rus": "Я просто странствующий бард. Я не достоин каких-либо подозрений! Воистину!"
+      "Rus": "Я просто странствующий бард. Я не тот, на кого тебе стоит тратить время! Воистину!"
```
а если персонаж женского пола?

xomachine commented on translations/texts/dialog/bounty.config.json:
```diff
@@ -690,7 +690,7 @@
     "Texts": {
       "Eng": "I am merely a wandering nomad. I am not worthy of any suspicion!",
-      "Rus": "Я просто странствующий кочевник. Я не достоин каких-либо подозрений! Воистину!"
+      "Rus": "Я просто странствующий кочевник. Я не тот, на кого тебе стоит тратить время! Воистину!"
```
тоже что и выше

vlad8599 committed:
Merge branch 'web-interface' into web-interface

xomachine commented on translations/texts/dialog/bounty.config.json:
```diff
@@ -859,7 +859,7 @@
     "Texts": {
       "Eng": "I have utilised fake handcuffs to trick you! I am still free!",
-      "Rus": "Я выкинул фальшивые наручники, чтобы обмануть тебя! Я все еще свободен!"
+      "Rus": "Я выкинул<self.pronoun.verbEnding> фальшивые наручники, чтобы обмануть тебя! Я все еще свободен!"
```
utilized - всё ещё не выкинуть а использовать

xomachine commented on translations/texts/dialog/bounty.config.json:
```diff
@@ -883,7 +883,7 @@
     "Texts": {
       "Eng": "I knew I could buy you out! Thanks for helping me get away!",
-      "Rus": "Я знал, что могу выкупить твою дою! Спасибо, что помог мне выбраться!"
+      "Rus": "Я знал<self.pronoun.verbEnding>, что могу подкупить тебя! Спасибо, что помог<self.pronoun.verb4Ending> мне выбраться!"
```
помог<player.pronoun.verb4Ending>

xomachine commented on translations/texts/dialog/bounty.config.json:
```diff
@@ -899,7 +899,7 @@
     "Texts": {
       "Eng": "I knew I could buy you out! Thanks for helping me get away, pal!",
-      "Rus": "Я знал, что могу выкупить твою дою! Спасибо, что помог мне выбраться, приятель!"
+      "Rus": "Я знал<self.pronoun.verbEnding>, что могу подкупить тебя! Спасибо, что помог<self.pronoun.verb4Ending> мне выбраться, приятель!"
```
тоже, что и выше

vlad8599 committed:
Перевел все, кроме последней фразы флорана(asdasd). Помогите с ней плиз.

xomachine commented on translations/texts/dialog/bounty.config.json:
```diff
@@ -1022,7 +1022,7 @@
     "Texts": {
       "Eng": "I shouldn't have volunteered for all those Miniknog test programs... I've never been the same since!",
-      "Rus": "Я не должен был добровольно участвовать во всех этих тестах Миникнога... Я никогда не буду прежним!"
+      "Rus": "Я не долж<self.pronoun.verb2Ending> был добровольно участвовать во всех этих тестах Министерства... Я никогда не буду прежн<self.pronoun.adj4Ending>!"
```
был<>

xomachine commented on translations/texts/dialog/bounty.config.json:
```diff
@@ -1192,7 +1192,7 @@
     "Texts": {
       "Eng": "IT'S NOT MY FAULT I'M MAD, THE MINIKNOG DID THINGS TO ME!",
-      "Rus": "Я НЕ ВИНОВАТ ЧТО Я БЕЗУМЕН, МИНИСТЕРСТВО СДЕЛАЛО ЭТО СО МНОЙ!"
+      "Rus": "Я НЕ ВИНОВАТ<self.pronoun.verbEnding> ЧТО Я БЕЗУМ<self.pronoun.verb2Ending>, МИНИСТЕРСТВО СДЕЛАЛО ЭТО СО МНОЙ!"
```
Тут окончания не прокатят, тк все буквы большие, а в окончаниях будут маленькие. надо перефразировать

xomachine commented on translations/texts/dialog/bounty.config.json:
```diff
@@ -1686,7 +1686,7 @@
     "Texts": {
       "Eng": "The secret truth is that everything is pixelly and flat! It can't just be me who sees it, can it?!",
-      "Rus": "Секрет в том, что все пиксельно и плоско! Я не могу быть всего лишь тем, кто это видит, не так ли ?!"
+      "Rus": "Секрет в том, что все пиксельно и плоско! Я не могу быть всего лишь т<self.pronoun.adj4Ending>, кто это видит, не так ли ?!"
```
Не могу же только я это видеть

vlad8599 replied:
тут нет подходящего окончания в базе(тем-той), добавьте или я не смогу это исправить.

xomachine replied:
тот вариант, который я предложил не требует специальных окончаний

xomachine commented on translations/texts/dialog/bounty.config.json:
```diff
@@ -1987,7 +1987,7 @@
     "Texts": {
       "Eng": "You saw through my disguise! It was I, <selfname>, all along!",
-      "Rus": "Ты раскрыл мою маскировку! Это был я, <selfname>, все время!"
+      "Rus": "Ты раскрыл<player.pronoun.verbEnding> мою маскировку! Это был я, <selfname>, все время!"
```
был<>

xomachine commented on translations/texts/dialog/bounty.config.json:
```diff
@@ -2001,7 +2001,7 @@
     "Texts": {
       "Eng": "You saw through my disguise! It was me, <selfname>, all along!",
-      "Rus": "Ты раскрыл мою маскировку! Это был я, <selfname>, все время!"
+      "Rus": "Ты раскрыл<player.pronoun.verbEnding> мою маскировку! Это был я, <selfname>, все время!"
```
был<>

xomachine commented on translations/texts/dialog/bounty.config.json:
```diff
@@ -1935,7 +1935,7 @@
     "Texts": {
       "Eng": "You have been caught by a jest! I used fake handcuffs to decieve you! Huzzah!",
-      "Rus": "Ты был пойман шуткой! Я использовал ненастоящие наручники и обманул тебя! Ура!"
+      "Rus": "Ты был<player.pronoun.verbEnding> пойман<player.pronoun.verbEnding> шуткой! Я использовал<self.pronoun.verbEnding> ненастоящие наручники и обманул<self.pronoun.verbEnding> тебя! Ура!"
```
Лучше наверное сказать "Ты повелся"

xomachine commented on translations/texts/dialog/bounty.config.json:
```diff
@@ -2115,7 +2115,7 @@
     "Texts": {
       "Eng": "You've been decieved by my fake handcuffs! I'm still as free as a bird!",
-      "Rus": "Ты был обманут моими фальшивыми наручниками! Я все еще свободен как птица!"
+      "Rus": "Ты был обманут<player.pronoun.verbEnding> моими фальшивыми наручниками! Я все еще свободен как птица!"
```
свобод<>

xomachine reviewed (commented):
Ещё немного усилий и будет хорошо

xomachine commented on translations/texts/dialog/peacekeeper.config.json:
```diff
@@ -33,7 +33,7 @@
     "Texts": {
       "Eng": "Floran iss good protector, will find prey!",
-      "Rus": ""
+      "Rus": "Флоран хороший Миротворец, он<self.pronoun.verbEnding> выследит добычу!"
```
тут нужно оставить он, ведь у флоранов нет пола

vlad8599 replied:
Ладно, запомню

vlad8599 committed:
Закончил часть файла peacekeeperconverse.config.json

xomachine commented on translations/texts/dialog/peacekeeperconverse.config.json:
```diff
@@ -51,7 +55,8 @@
     "Texts": {
-      "Eng": "Captain Noble has me working on finding some new cases for you, Officer <entityname>!"
+      "Eng": "Captain Noble has me working on finding some new cases for you, Officer <entityname>!",
+      "Rus": "Капитан Нобель заставил меня работать над поиском каких-нибудь новых дел для вас, Офицер <entityname>!"
```
лучше убрать "каких-нибудь"

xomachine commented on translations/texts/dialog/peacekeeperconverse.config.json:
```diff
@@ -183,7 +199,8 @@
     "Texts": {
-      "Eng": "Howdy there, Officer <entityname>! Do you know if Captain Noble is single? I'm askin' for... A friend."
+      "Eng": "Howdy there, Officer <entityname>! Do you know if Captain Noble is single? I'm askin' for... A friend.",
+      "Rus": "Эй, там, Офицер <entityname>! Ты слышал, что Капитан Нобель одинок? Я говорю о... другом."
```
Говорящий спрашивает, одинок ли Капитан  Нобель. и признаётся, что ищет друга

yasupergey replied:
> Говорящий спрашивает, одинок ли Капитан Нобель. и признаётся, что ищет друга
Говорящий спрашивает нет ли пары у Капитана и говорит что спрашивает для друга

xomachine replied:
@Morden-not-a-modern Ваша правда!

xomachine commented on translations/texts/dialog/peacekeeperconverse.config.json:
```diff
@@ -205,7 +223,8 @@
     "Texts": {
-      "Eng": "Howdy! I'm Officer <selfname>, and I'm ready to bust up some bad guys!"
+      "Eng": "Howdy! I'm Officer <selfname>, and I'm ready to bust up some bad guys!",
+      "Rus": "Приветствую! Я Офицер <selfname>, и я готов отправить за решетку плохих парней!"
```
а если говорящий женского пола?

xomachine commented on translations/texts/dialog/peacekeeperconverse.config.json:
```diff
@@ -216,7 +235,8 @@
     "Texts": {
-      "Eng": "I am determined to bring peace to this sector once more!"
+      "Eng": "I am determined to bring peace to this sector once more!",
+      "Rus": "Я полон решимости принести мир в этот сектор еще раз!"
```
пол<>

xomachine commented on translations/texts/dialog/peacekeeperconverse.config.json:
```diff
@@ -271,7 +295,8 @@
     "Texts": {
-      "Eng": "I joined up with the Peacekeepers mostly because of the Captain's sheer enthusiasm. Such charisma!"
+      "Eng": "I joined up with the Peacekeepers mostly because of the Captain's sheer enthusiasm. Such charisma!",
+      "Rus": "Я присоединился к Миротворцам в основном из-за того, что Капитан прямо сияет энтузиазмом. Такая харизма!"
```
Присоединил<>

vlad8599 commented:
Я уже завтра исправлю, мне надо хотя бы этот файл до ума довести(еще не все реплики переведены). А потом можно и рода исправлять.

vlad8599 committed:
Merge 1ccc25424657d128200d222faac22cd6bdcc1b43 into web-interface

vlad8599 committed:
Merge eccd616246ff056c380d19ff3fdf2c10bc22d156 into web-interface

vlad8599 committed:
Merge fa8f4e566530918154a01df7f8f570d457d7f4dd into web-interface

vlad8599 committed:
Merge 3d2d9754292a29e04da13a0f04db4be9de416fc5 into web-interface

vlad8599 committed:
Merge e55b02b1be4898140f87903329918f716e630a02 into web-interface

vlad8599 committed:
Merge 0704a995a16e1ccedaa519a4c10bf8da88f78657 into web-interface

vlad8599 committed:
Merge a4143a63ba4ab00433917094e4ccbcc68c5e9106 into web-interface

vlad8599 committed:
Merge 3daa4a6e53fc1d356202eeaab37ff05ec8911514 into web-interface

xomachine committed:
Merge branch 'web-interface' into web-interface

xomachine committed:
Merge branch 'web-interface' into web-interface

vlad8599 committed:
Merge 7c2c6995ff8f015eadc15e378a180f9bf8ac5388 into web-interface

vlad8599 committed:
Обновление translations/texts/dialog/bounty.config.json

vlad8599 committed:
исправлены ошибки перевода

xomachine committed:
Update bounty.config.json

сделал выкрики безумцев гендерно-нейтральными
исправил passive voice (to be rambled != to ramle)
исправил потерянные и лишние пробелы

vlad8599 committed:
Merge 22bfae7fd7c844a577d6faa086d17f8faa3e68cf into web-interface

vlad8599 committed:
Обновление translations/texts/dialog/peacekeeper.config.json

vlad8599 committed:
Обновление translations/texts/dialog/peacekeeper.config.json

vlad8599 committed:
Merge 006b6a7421c347dbc30dee05c9d71544012694e6 into web-interface

vlad8599 committed:
Закончил файл peacekeeperconverse.config.json

vlad8599 commented:
Фух, кажется, я наконец закончил с этим пулл реквестом. Исправил то, о чем говорилось, плюс закончил второй файл.

vlad8599 committed:
Мелкое исправление, не заметил сначала

vlad8599 committed:
Добавлен временный вариант перевода "Asdasd"

vlad8599 committed:
Начал работу над bountytarget.config.json

xomachine commented on translations/texts/dialog/peacekeeperconverse.config.json:
```diff
@@ -571,7 +595,8 @@
     "Texts": {
-      "Eng": "The Captain has me investigating local criminal activity. These gangs sure do pick the weirdest gang names, huh?"
+      "Eng": "The Captain has me investigating local criminal activity. These gangs sure do pick the weirdest gang names, huh?",
+      "Rus": "Капитан заставляет меня расследовать местную преступную деятельность. Эти банды наверняка выбирают самые странные названия, правда?"
```
явно, определённо, но не наверяка

наверняка добавляет некий оттенок неуверенности говорящего, а этого тут нет

xomachine commented on translations/texts/dialog/peacekeeperconverse.config.json:
```diff
@@ -560,7 +583,8 @@
     "Texts": {
-      "Eng": "The Captain has informed me of your prowess, Officer <entityname>."
+      "Eng": "The Captain has informed me of your prowess, Officer <entityname>.",
+      "Rus": "Капитан проинформировал меня о вашей доблести, Офицер <entityname>."
```
Вашей

xomachine commented on translations/texts/dialog/peacekeeperconverse.config.json:
```diff
@@ -670,7 +703,8 @@
     "Texts": {
-      "Eng": "Well ruffle my feathers - you're Officer <entityname>! The Captain has such kind words to say about you!"
+      "Eng": "Well ruffle my feathers - you're Officer <entityname>! The Captain has such kind words to say about you!",
+      "Rus": "Хорошо же вы потрепали мне перья, Офицер <entityname>! У Капитана есть столь любезные слова для вас!"
```
что-то тут не так. персонаж не сражался с игроком но говорит о потрёпанных перьях

странностей быть не должно

vlad8599 replied:
Тогда что тут больше подойдет? Явно речь идет о потрепанных перьях, очевидно же.

xomachine replied:
https://idioms.thefreedictionary.com/ruffle+feathers

вот подсказка

xomachine commented on translations/texts/dialog/peacekeeperconverse.config.json:
```diff
@@ -692,7 +727,8 @@
     "Texts": {
-      "Eng": "Without a Matter Manipulator, there's only so much field work I can do. Not like you, Officer <entityname>!"
+      "Eng": "Without a Matter Manipulator, there's only so much field work I can do. Not like you, Officer <entityname>!",
+      "Rus": "Без Манипулятора Материи это все, что я могу сделать. Не то что вы, Офицер <entityname>!"
```
Вы

xomachine commented on translations/texts/dialog/peacekeeperconverse.config.json:
```diff
@@ -703,7 +739,8 @@
     "Texts": {
-      "Eng": "Word around the station is that you're a real sharp-shooter, Officer <entityname>!"
+      "Eng": "Word around the station is that you're a real sharp-shooter, Officer <entityname>!",
+      "Rus": "По станции ходит слух, что вы настоящий снайпер, Офицер <entityname>!"
```
Вы

xomachine commented on translations/texts/dialog/peacekeeperconverse.config.json:
```diff
@@ -714,7 +751,8 @@
     "Texts": {
-      "Eng": "You're Officer <entityname>?! I've heard so much about you from the Captain! You look different to how I imagined..."
+      "Eng": "You're Officer <entityname>?! I've heard so much about you from the Captain! You look different to how I imagined...",
+      "Rus": "Вы Офицер <entityname>?! Я так много слышал<self.pronoun.verbEnding> о вас от Капитана! Вы выглядите совершенно иначе, чем я представлял<self.pronoun.verbEnding> себе..."
```
Вас

xomachine commented on translations/texts/dialog/peacekeeperconverse.config.json:
```diff
@@ -604,7 +631,8 @@
     "Texts": {
-      "Eng": "There's a few new cases I've been researching just for you, Officer <entityname>."
+      "Eng": "There's a few new cases I've been researching just for you, Officer <entityname>.",
+      "Rus": "Вот несколько новых случаев, которые я откопал специально для вас, Офицер <entityname>."
```
Вас

xomachine commented on translations/texts/dialog/peacekeeperconverse.config.json:
```diff
@@ -494,7 +511,8 @@
     "Texts": {
-      "Eng": "Officer <entityname>? The Captain told me you're the one who has helped get this operation running again. You have my admiration."
+      "Eng": "Officer <entityname>? The Captain told me you're the one who has helped get this operation running again. You have my admiration.",
+      "Rus": "Офицер <entityname>? Капитан сказал мне, что именно вы помогли продолжить работу над этой операцией. Я восхищен вами."
```
Вы...Вами

xomachine commented on translations/texts/dialog/peacekeeperconverse.config.json:
```diff
@@ -461,7 +475,8 @@
     "Texts": {
-      "Eng": "My role is to gather intel on our local gangs. It's your job, Officer <entityname>, to take the fight to them!"
+      "Eng": "My role is to gather intel on our local gangs. It's your job, Officer <entityname>, to take the fight to them!",
+      "Rus": "Моя задача - сбор информации о местных бандах. Драться с ними - ваша работа, Офицер <entityname>!"
```
Ваша

xomachine commented on translations/texts/dialog/peacekeeperconverse.config.json:
```diff
@@ -439,7 +451,8 @@
     "Texts": {
-      "Eng": "Keep your chin up, Officer -  It's a tough job but I believe in you!"
+      "Eng": "Keep your chin up, Officer -  It's a tough job but I believe in you!",
+      "Rus": "Выше голову, Офицер - это сложная работа, но я верю в вас!"
```
Вас

xomachine commented on translations/texts/dialog/peacekeeperconverse.config.json:
```diff
@@ -329,7 +331,8 @@
     "Texts": {
-      "Eng": "I'm gonna be real honest with you, Officer <entityname>. I joined up mostly because the Captain is mighty handsome."
+      "Eng": "I'm gonna be real honest with you, Officer <entityname>. I joined up mostly because the Captain is mighty handsome.",
+      "Rus": "Я буду откровен<self.pronoun.verb2Ending> с вами, Офицер <entityname>. Я присоединил<self.pronoun.verb3Ending> к вам в основном потому, что Капитан очень красивый."
```
Вами

xomachine commented on translations/texts/dialog/peacekeeperconverse.config.json:
```diff
@@ -318,7 +319,8 @@
     "Texts": {
-      "Eng": "I'll work my way up the Peacekeeper chain until I can take on the Miniknog myself!"
+      "Eng": "I'll work my way up the Peacekeeper chain until I can take on the Miniknog myself!",
+      "Rus": "Я буду пробивать себе путь вверх по цепочке Миротворца, пока не смогу однажды лично померятся силой с Министерством!"
```
лучше сказать по карьерной лестнице Миротворцев

xomachine commented on translations/texts/dialog/peacekeeperconverse.config.json:
```diff
@@ -307,7 +307,8 @@
     "Texts": {
-      "Eng": "I sure would like to get my OWN desk one day."
+      "Eng": "I sure would like to get my OWN desk one day.",
+      "Rus": "Я уверен<self.pronoun.verbEnding>, что получу однажды свой собственный стол."
```
что однажды получу свой СОБСТВЕННЫЙ стол

xomachine commented on translations/texts/dialog/bountytarget.config.json:
```diff
@@ -259,7 +259,7 @@
     "Texts": {
       "Eng": "Did you cuff me jus' by fightin' me? How does that work?",
-      "Rus": ""
+      "Rus": "Ты надел на меня наручники, просто сражаясь со мной? Как это работает?"
```
надел<>

xomachine commented on translations/texts/dialog/peacekeeper.config.json:
```diff
@@ -154,7 +154,7 @@
     "Texts": {
       "Eng": "asdasd",
-      "Rus": ""
+      "Rus": "*злобное шипение*"
```
лучше пока оставить без перевода, кроме того таким образом звукоподражания мы не переводим. если это звукоподражание, то и без знания английского можно прочитать его

xomachine commented on translations/texts/dialog/peacekeeperconverse.config.json:
```diff
@@ -572,7 +572,7 @@
     "Texts": {
       "Eng": "The Captain exudes confidence. I one day hope to be even half as inspirational as he is!",
-      "Rus": "Капитан излучает уверенность. Я надеюсь в один день стать хотя бы вполовину так<self.pronoun.adj4Ending> же вдохновляющим, как он!"
+      "Rus": "Капитан излучает уверенность. Я надеюсь в один день стать хотя бы вполовину так<self.pronoun.adj4Ending> же вдохновляющ<self.pronoun.adj4Ending>, как он!"
```
однажды

vlad8599 committed:
Merge 305013ffef902058bfa8681ed36a4ff0643974ac into web-interface

vlad8599 committed:
Исправления

vlad8599 committed:
Обновление translations/texts/dialog/bountytarget.config.json

vlad8599 commented:
Готово.

xomachine committed:
Финишная прямая

xomachine committed:
Update peacekeeperconverse.config.json

xomachine committed:
Update peacekeeper.config.json

xomachine committed:
Update bounty.config.json
